CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ if(APPLE)
     -mmacosx-version-min=${OSX_VERSION_MIN}
   )
 
-  set(CXX_COMPILE_FLAGS "${CXX_COMPILE_FLAGS} -std=c++11 -stdlib=libc++")
+  set(CXX_COMPILE_FLAGS "${CXX_COMPILE_FLAGS} -std=c++14 -stdlib=libc++")
```

Comment:
Woot, sneaking this in here for `std::make_unique`, but it's time we moved otherwise.